<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommLogForm</class>
 <widget class="QDialog" name="CommLogForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>702</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0">
       <item>
        <widget class="QPushButton" name="btnDateTime">
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>PushButton</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="hlHour">
       <item>
        <widget class="QLabel" name="lblSelHours">
         <property name="font">
          <font>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Hour:</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QCheckBox" name="cbAllHours">
       <property name="text">
        <string>All</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0">
     <item>
      <widget class="QPushButton" name="btnPrintSum">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Print Summary ...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPrintDtl">
       <property name="maximumSize">
        <size>
         <width>150</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Print Detail ...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbExpand">
       <property name="text">
        <string>Expand</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeView" name="tvCommLog"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
